apps/frontend/src/App.tsx

```diff
@@ -143,6 +148,10 @@ const router = createBrowserRouter([
             element: <Class.Grades />,
             path: "grades",
           },
+          {
+            element: <CatalogRatings />,
```

Comment:
You can see how the components are defined as `<Class.Component />` which is what you should have here as it is part of the larger `<Class />` component:
```ts
element: <Class.Ratings />
```